lib/gameState/CGameState.cpp

```diff
@@ -566,17 +566,20 @@ void CGameState::initNewGame(const IMapService * mapService, bool allowSavingRan
 		{
 			try
 			{
-				auto path = VCMIDirs::get().userCachePath() / "RandomMaps";
+				auto path = VCMIDirs::get().userDataPath() / "Maps" / "RandomMaps";
```

Comment:
So I guess we now save random maps inside "Maps" directory.
Not against the change, considering that we allow subdirectories now.

Reply:
Yes, exactly. But this folder is hidden and only shown, when using the "show random" button (then all other folders/maps are hidden).